tslog.sh

```diff
@@ -47,6 +49,14 @@ while read THREAD; do
 		echo "$TD $TSC STACK $STACK"
 	done > tslog.thread.$THREAD
 done < threads
+if [ ! -f tslog.thread.0x0 ]; then
```

Comment:
We could handle this case by processing what we have (i.e. kernel tslog).  It wouldn't give us a complete picture of the boot process, but might still be better than nothing?